static/app/types/index.tsx

```diff
@@ -37,6 +55,19 @@ declare global {
      */
     __pipelineInitialData: PipelineInitialData;
 
+    /**
+     * This allows our server-rendered templates to push configuration that should be
+     * run after we render our main application.
+     *
+     * An example of this is dynamically importing the `passwordStrength` module only
+     * on the organization login page.
+     */
+    __onSentryInit:
+      | OnSentryInitConfiguration[]
+      | {
```

Comment:
Shouldn't this actually be an intersection?

Reply:
Nope, it's an array of configs if our templated JS runs first, otherwise it will be an object that has the `push` interface (where it partially mocks an array).